src/msg/async/Event.cc

```diff
@@ -331,11 +332,18 @@ int EventCenter::process_events(int timeout_microseconds)
 
   ldout(cct, 10) << __func__ << " wait second " << tv.tv_sec << " usec " << tv.tv_usec << dendl;
   vector<FiredFileEvent> fired_events;
+
+  rcu_thread_offline();
```

Comment:
similarly here;  I understand this logic pretty well from NEBR (an epoch-based reclaimation variant), but find the offline() online() notation surprising and might not have been 100% sure of the behavior if I were just encountering it